src/main/java/io/meeds/qa/ui/pages/page/factory/space/ManageSpacesPage.java

```diff
@@ -78,7 +77,7 @@ public class ManageSpacesPage extends GenericPage {
   @FindBy(xpath = "(//div[@name='inviteMembers']//input)[1]")
   private TextBoxElementFacade inviteUserInput;
 
-  @FindBy(xpath = "(//div[contains(text(), 'Move after')])[2]")
+  @FindBy(xpath = "//div[contains(text(), 'Move after')]")
   private BaseElementFacade    moveAfterAppButton;
```

Comment:
as discussed, we may need to be more precise here to make sure to search for an active menu button instead of a hidden one.

Reply:
done